Example/Tests/GINISupportedFormatsViewControllerTests.swift

```diff
@@ -12,6 +12,10 @@ import XCTest
 final class GINISupportedFormatsViewControllerTests: XCTestCase {
     
     var supportedFormatsViewController = SupportedFormatsViewController(style: .plain)
+    let initialGiniConfiguration:GiniConfiguration = {
+        GiniConfiguration.sharedConfiguration.fileImportSupportedTypes = .pdf_and_images
```

Comment:
Why do you need that? The whole point of the shared instance injection is to NOT alter the regular shared instance. 

Why not just have:

`let initialGiniConfiguration = GiniConfiguration.sharedConfiguration`